.github/workflows/coverage.yml

```diff
@@ -39,7 +39,16 @@ jobs:
         run: cargo nextest run --workspace --exclude topos-sequencer-subnet-runtime
 
       - name: Run grcov
-        run: grcov . --binary-path target/debug/ -s . -t lcov --branch --ignore-not-existing --ignore '../**' --ignore '/*' -o coverage.lcov
+        run: |
+          grcov . --binary-path target/debug/ -s . \
+            -t lcov \
+            --branch \
+            --ignore-not-existing \
+            --ignore 'crates/topos-api/src/grpc/generated/*' \
+            --ignore 'crates/topos-test-sdk/src/grpc/behaviour/*' \
```

Comment:
Shouldn't all of `topos-test-sdk` be ignored? Do we care about coverage for test utilities?

Reply:
Maybe, but as it is code that are used to build tests maybe we should keep them to spot unused or unreachable code blocks.

Reply:
Fair enough.